### ISSUE TITLE
How to install using cocoapod.

### ISSUE THREAD

**Issue by rvishal1991** [teambition/RRuleSwift#7]
How can install using Cocoapod.
I am unable to run by installing Manually.
Thanks in advance.

**Comment by guidove** [comment]
For manual install, follow the Alamofire manual install procedure.

**Comment by erychagov** [comment]
https://github.com/teambition/RRuleSwift/pull/9

**Comment by kohlab** [comment]
Yes, please support installation via CocoaPods and update the docs.

**Comment by mpaillet3821** [comment]
Same problem here, please support Cocoapods
